contracts/Vouchers.sol

```diff
@@ -0,0 +1,262 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+pragma solidity 0.7.6;
+
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "./interfaces/IVoucherKernel.sol";
+import "./interfaces/ICashier.sol";
+import "./interfaces/IVouchers.sol";
+
+//preparing for ERC-1066, ERC-1444, EIP-838
+
+/**
+ * @title Vouchers implemented as ERC-721
+ */
+// TODO: inherit from OZ ERC721 and remove state vars and local implementations of IERC721
```

Comment:
I feel we can remove this TODO as the `ERC721` of OZ is used now.

Reply:
Removed. Will be included in next commit.